front-end-admin/src/api/ui-management.ts

```
@@ -0,0 +1,176 @@
+): UseQueryResult<GetUserDetailsResponse> => {
+  return useQuery({
+    queryKey: ['get_click_trend'],
+    queryFn: async () => {
+      try {
+        const response = await authFetch.get<GetUserDetailsResponse>(
+          `/user-interaction-service/api/miss-click-rate-by-button`
+        )
+        console.log(response.data)
+        return response?.data.data
+      } catch (error) {
+        throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)